ext/standard/basic_functions.c

```diff
@@ -418,10 +418,27 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_array_keys, 0, 0, 1)
 	ZEND_ARG_INFO(0, strict)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_INFO(arginfo_array_key_first, 0)
+	ZEND_ARG_INFO(0, arg) /* ARRAY_INFO(0, arg, 0) */
```

Comment:
Why is the comment ARRAY_INFO, instead of just using ``ZEND_ARG_ARRAY_INFO`` ?

Reply:
Since we don't use the typed arginfo parameters, yet. @nikic recently suggested to stick with that for now to avoid duplicate checks, IIRC.

Reply:
I adopted it from existing functions for consistency so I guess keep it?